src/components/DeleteConfirmationModal/DeleteConfirmationModal.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { FunctionComponent } from 'react';
+import { Button, Checkbox, List, ListItem, Modal, ModalVariant } from '@patternfly/react-core';
+import { getConfig } from '../../utilities/platformServices';
+import Cookies from 'js-cookie';
+
+interface DeleteConfirmationModalProps {
+  handleModalToggle: () => void;
+  successCallback: () => void;
+  isOpen: boolean;
+  name: string;
+  uuid: string;
+}
+
+const DeleteConfirmationModal: FunctionComponent<DeleteConfirmationModalProps> = ({
+  handleModalToggle,
+  successCallback,
+  isOpen,
+  name,
+  uuid
+}) => {
+  const [confirmed, setConfirmed] = React.useState(false);
+
+  const deleteManifest = () => {
+    const jwtToken = Cookies.get('cs_jwt');
+    const { rhsmAPIBase } = getConfig();
+    fetch(`${rhsmAPIBase}/management/v1/allocations/${uuid}?force=true`, {
+      method: 'DELETE',
+      headers: { Authorization: `Bearer ${jwtToken}` },
+      mode: 'cors'
+    }).then((response) => {
+      if (response.status != 200) {
+        throw new Error(
+          `Status Code ${response.status}.  Error deleting manifest: ${response.statusText}.`
+        );
+      }
+      successCallback();
+    });
+  };
+
+  const handleConfirmation = () => {
+    deleteManifest();
+    handleModalToggle();
```

Comment:
I don't know if leaving the confirmation popup open after confirming would make sense, but I think you have a point that it should have some sort of feedback.  I think it will look a bit more natural when the table has to reload after the delete per your other comment, but it could still probably use something else.  Do we even have another way to notify people of things?  We might need a way to deliver success/failure messages in a floating box in a corner of the screen or something.

Reply:
Aha here - https://www.patternfly.org/v4/components/alert#alert-timeout This may be all we need. The above package uses redux and is a bit overly complicated.